datalad_deprecated/resources/website/assets/js/main.js

```diff
@@ -585,6 +691,38 @@ function directory(jQuery, md5) {
   return table;
 }
 
+// https://stackoverflow.com/questions/47207355/copy-to-clipboard-using-jquery/47207504
+function copyToClipboard(text) {
+  var $temp = $('<input>');
+  $('body').append($temp);
+  $temp.val(text).select();
+  document.execCommand('copy');
+  $temp.remove();
+}
+
+function getRenderers(filename) {
+  validRenderers = []
+
+  EXTERNAL_RENDERERS.forEach(function(renderer) {
+    var regex = new RegExp(renderer.regex);
+    if (regex.test(filename)) {
+      // add rendering option
+      // https://stackoverflow.com/questions/14780350/convert-relative-path-to-absolute-using-javascript
+      var link = document.createElement("a");
+      link.href = filename;
+      link.protocol = "https";
+
+      validRenderers.push({
+        "name": renderer.name,
+        "url": renderer.endpoint + link.href,
```

Comment:
eventually we should make it a proper "interpolation" or alike, but for now as long as data structure defining renderers is "built-in", ok - let it be just an append until it can no longer ;)